src/resources/app/pages/ViewCourse.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="screen mx-16">
+    <div
+      class="h-16"
+    ></div>
+    <div
+      class="flex flex-row flex-wrap items-start space-x-10"
+    >
+      <div 
+        class="m-0"
+      />
+      <div v-for="post in posts" v-bind:key="post.id">
+        <Card 
+        :course-name="post.name"
+        :room-num="post.room"
+        :teacher-name="post.createdBy"
+
+        id="course-card"
+        class="mb-10"
+      />
+      </div>
+      
+
+      <Add 
+        id="add-course-card"
+        class="mb-10"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import Card from "../components/CourseCard.vue";
+import Add from "../components/AddCourseCard.vue";
```

Comment:
nit:
1. order the imports lexicographically
2. use the file name as the component name e.g. `import CourseCard from "..."`